src/components/common/EntityInfo.vue

```diff
@@ -127,4 +149,7 @@ export default class EntityInfo extends Mixins(EnumMixin) {
   color: $gray9;
   font-weight: bold;
 }
+.staff-comments{
+  margin-left: -12px;
+}
```

Comment:
Does the alignment of the staff comment component work ok when you change the screen width?

Reply:
I recommend (but don't insist on) just adding class `ml-n3` to the template... IMO, layout changes like this are easier to manage if they are in the template.